pom.xml

```diff
@@ -189,8 +189,8 @@
     <protobuf-maven-plugin.version>0.6.1</protobuf-maven-plugin.version>
     <cyclonedx.version>2.8.0</cyclonedx.version>
 
-    <spotbugs.version>4.2.1</spotbugs.version>
-    <spotbugs-plugin.version>4.2.0</spotbugs-plugin.version>
+    <spotbugs.version>4.8.6</spotbugs.version>
+    <spotbugs-plugin.version>4.8.6.2</spotbugs-plugin.version>
```

Comment:
Why not 4.9.x?

Reply:
4.8.6 fits the building with JDK21, thus it's chosen.

Or do you suggest we choose a more up-to-date version?

Reply:
Let's use 4.9.x.  Apache Hadoop has been [updated to 4.9.x](https://github.com/apache/hadoop/pull/8028).  It is working fine.